tasks/prime.py

```diff
@@ -347,3 +347,34 @@ def create_move_task_order(self):
 
         if success:
             self.set_prime_data(PrimeObjects.MOVE_TASK_ORDER, mto_data)
+
+    @tag(PrimeObjects.MTO_SERVICE_ITEM.value, "updateMTOServiceItemStatus")
+    @task
+    def update_mto_service_item_status(self):
+        mto_shipment = self.get_random_data(PrimeObjects.MTO_SHIPMENT)
+        # if we don't have an mto shipment we can't run this task
+        if not mto_shipment:
+            return
+
+        # if we don't have any service items created for mto shipment can't run this task
+        if mto_shipment.get("mtoServiceItems") is None:
+            return
+
+        payload = self.fake_request("/service-items/{mtoServiceItemID}/status", "patch")
+        mto_service_item = mto_shipment["mtoServiceItems"][0]
+        headers = {"content-type": "application/json", "If-Match": mto_service_item["eTag"]}
+
+        resp = self.client.patch(
+            support_path(f"/service-items/{mto_service_item['id']}/status"),
+            name=support_path("/mto-shipments/:mtoShipmentID/status"),
```

Comment:
looks like a copy paste error here ☺️ - should be "/service-tems/{mtoServiceItemID}/status" < we're going to go back at make the curly braces standard here since that's what it looks like in the API YAML file

Reply:
I've updated this! Thanks for catching this!